analyzer/src/test/kotlin/managers/utils/PubCacheReaderTest.kt

```diff
@@ -30,11 +32,15 @@ import org.ossreviewtoolkit.utils.test.createSpecTempDir
 private const val PACKAGE_NAME = "non-existing-package"
 private const val PACKAGE_VERSION = "0.0.0"
 private const val RESOLVED_REF = "0000000000000000000000000000000000000000"
+private val RELATIVE_PATH = ".."
```

Comment:
This should become `const` to silence he *detekt* issue.

Reply:
And while you're it, maybe you could also fix a commit message nit, and write capitalized "Add" after the colon in the title.

Reply:
done